labseu2-workout-tracker/server
# View exercise details

##  What does this PR do
This PR allows you to find individual exercise by ID

##  Description of task to be completed
Pulls out individual exercise from exercise DB using end point /exercises/:id

##  How should this manually be tested

To test use endpoint /exercises/:id where :id equals number
Numbers 1 - 899 should return full exercise in the following JSON format with 200 status code

`{
    "id": 500,
    "exercise_name": "Power Snatch",
    "exercise_ratings": "9.2",
    "description": "Begin with a loaded barbell on the floor. The bar should be close to or touching the shins, and a wide grip should be taken on the bar. The feet should be directly below the hips, with the feet turned out as needed. Lower the hips, with the chest up and the head looking forward. The shoulders should be just in front of the bar. This will be the starting position. Begin the first pull by driving through the front of the heels, raising the bar from the ground. The back angle should stay the same until the bar passes the knees. Transition into the second pull by extending through the hips knees and ankles, driving the bar up as quickly as possible. The bar should be close to the body. At peak extension, shrug the shoulders and allow the elbows to flex to the side. As you move your feet into the receiving position, a slightly wider position, pull yourself below the bar as you elevate the bar overhead. The bar should be received in a partial squat. Continue raising the bar to the overhead position, receiving the bar locked out overhead. Return to a standing position with the weight over head.",
    "difficulty": "Expert",
    "video": "https://videos.bodybuilding.com/video/mp4/78000/78342m.mp4",
    "picture_one": "https://www.bodybuilding.com/exercises/exerciseImages/sequences/703/Male/l/703_4.jpg",
    "picture_two": "https://www.bodybuilding.com/exercises/exerciseImages/sequences/703/Male/l/703_2.jpg",
    "type": "Olympic Weightlifting",
    "muscle": "Hamstrings",
    "equipment": "Barbell",
    "created_at": "2019-08-23T11:26:47.206Z",
    "updated_at": "2019-08-23T11:26:47.206Z"
}`

Will return 400 error with Invalid ID response for any id that does not have an exercise.
WIll return 500 error for internal server errors


##  What are the relevant Trello board stories

https://trello.com/c/RDI7g8gt/62-as-a-user-i-want-to-be-able-to-view-the-details-of-an-exercise-single-exercise

##  Screenshots (If appropriate)


##  Questions (If any):
